runtime/src/main/java/org/corfudb/runtime/view/StreamsView.java

```diff
@@ -142,9 +145,17 @@ public long append(@Nonnull Set<UUID> streamIDs, @Nonnull Object object,
                 // If we're here, we succeeded, return the acquired token
                 return tokenResponse.getTokenValue();
             } catch (OverwriteException oe) {
+                int sizeOfWrite = 0;
```

Comment:
This calculation should only be done if the log is set to output warn level messages